pkgs/servers/monitoring/prometheus/sachet.nix

```diff
@@ -0,0 +1,25 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "sachet";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "messagebird";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-zcFViE1/B+wrkxZ3YIyfy2IBbxLvXOf8iK/6eqZb1ZQ=";
+  };
+
+  vendorSha256 = null;
+
+  meta = with lib; {
+    description = "Sachet is an SMS alerting tool for the Prometheus Alertmanager.";
```

Comment:
Don't include a period, and don't repeat the name of the package -- see [the manual](https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes).